server/package.json

```diff
@@ -15,6 +15,7 @@
     "@types/helmet": "^0.0.45",
     "compression": "^1.7.4",
     "cookie-parser": "^1.4.4",
+    "cors": "^2.8.5",
```

Comment:
Is this package required for the resolution of #694?

Reply:
> Is this package required for the resolution of #694?

I think we had added that with @p-netm when troubleshooting when I had trouble setting `tiny-server` up.  we later found out the problem was `REACT_APP_DISABLE_LOGIN_PROTECTION=false` had been set to true